source/client.js

```diff
@@ -13,7 +15,11 @@ const pageDetails = parse(window.location.search)
 
 render(
   <div>
```

Comment:
Kinda makes the care displayed here:
https://github.com/mixpanel/mixpanel-js/blob/master/build/mixpanel.cjs.js#L33
Pointless